typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -10,7 +10,7 @@ export type ChainMetadata = {
   paginate?: RpcPagination;
   // The CoinGecko API expects, in some cases, IDs that do not match
   // ChainNames.
-  coinGeckoId?: string;
+  gasCurrencyCoinGeckoId?: string;
```

Comment:
For chains that use Ethereum to pay for gas but still have their own native token (like Optimism with OP, Arbitrum will have a token but it hasn't release yet, or Aurora), I wanted to avoid confusion. Without context I'd assume Optimism's `coinGeckoId` was for the OP token, not the ETH token